cmd/sonobuoy/app/cp.go

```diff
@@ -0,0 +1,51 @@
+/*
+Copyright 2018 Heptio Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package app
+
+import (
+	"os"
+
+	ops "github.com/heptio/sonobuoy/cmd/sonobuoy/app/operations"
+	"github.com/heptio/sonobuoy/pkg/errlog"
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+)
+
+var path string
+
+func init() {
+	cmd := &cobra.Command{
+		Use:   "cp",
+		Short: "Copies the results to a specified path",
+		Run:   copyResults,
+	}
+	cmd.PersistentFlags().StringVar(
+		&path, "path", "./",
+		"TBD: location to output",
+	)
+	// TODO: Other Options.?.?
```

Comment:
Any objections to leaving a username along with the TODO? It does not indicate ownership, simply someone to talk to about it.

```TODO(chuckha)```

I have found it also helps me write full sentences.

Reply:
I figure we'll sort in the backlog.